--- phonon-4.11.1/phonon/CMakeLists.txt.orig	2019-09-26 12:29:38.000000000 +0200
+++ phonon-4.11.1/phonon/CMakeLists.txt	2025-06-09 20:10:58.782368540 +0200
@@ -67,7 +67,7 @@ set(phonon_LIB_SRCS
    ${phonon_VGO_SRCS}
    )
 
-ecm_create_qm_loader(phonon_LIB_SRCS libphonon_qt)
+ecm_create_qm_loader(phonon_LIB_SRCS libphonon_qt${QT_MAJOR_VERSION})
 
 option(PHONON_NO_PLATFORMPLUGIN "Do not use any platform plugin")
 if (PHONON_NO_PLATFORMPLUGIN)
